fv3core/stencils/d_sw.py

```diff
@@ -307,15 +316,15 @@ def heat_source_from_vorticity_damping(
         rdy (in): radius of Earth multiplied by y-direction gridcell width
         heat_source (out): heat source from vorticity damping
             implied by energy conservation
-        diss_est (out): dissipation estimate, only calculated if
+        dissipation_estimate (out): dissipation estimate, only calculated if
             calculate_dissipation_estimate is 1
         kinetic_energy_fraction_to_damp (in): according to its comment in fv_arrays,
             the fraction of kinetic energy to explicitly damp and convert into heat.
             TODO: confirm this description is accurate, why is it multiplied
             by 0.25 below?
-        calculate_dissipation_estimate (in): If 1, calculate dissipation estimate.
```

Comment:
Since this is being deleted, is there somewhere high-level we can transfer it? Whether in a docstring somewhere, or in a comment or document where we describe namelist variables. If not that's OK.

Reply:
it's only deleted in favor of using the namelist value as an external. when this is an object, we can use this name. I'll add a comment about it